tests/kinetic_driving_force/screen-output

```diff
@@ -0,0 +1,44 @@
+
+Loading shared library <./libkinetic_driving_force.debug.so>
```

Comment:
Please also add the `statistics` file to the output directory of the test, it will make the test comparison a bit more strict.